### PR TITLE
feat(#minor); Sushiswap; Added harmony to Sushiswap

### PR DESCRIPTION
**Pull request by steegecs**
**Context:**
Harmony is now supported so we are now adding Sushiswap Harmony.

### CODE REVIEW

**Review by dmelotik — APPROVED** [review]
lgtm, do you have a test deployment that is syncing properly?

File: subgraphs/uniswap-forks/protocols/sushiswap/config/networks/harmony/harmony.ts
Comment (jaimehgb) [inline subgraphs/uniswap-forks/protocols/sushiswap/config/networks/harmony/harmony.ts:78]: wBNB?

**Review by jaimehgb — APPROVED** [review]
lgtm!

**Comment by steegecs** [comment]
> lgtm, do you have a test deployment that is syncing properly?

Yeah - Here is the location. It has about 20% lower TVL which I am looking into. I think this might be an error on the sushi analytics side because the use such a low minimum liquidity threshold of about $1000 whereas I use $5000. I am testing if the TVL will match more closely by adjusting this. 

https://thegraph.com/hosted-service/subgraph/messari/sushiswap-harmony
https://analytics-harmony.sushi.com/

**Comment by dmelotik** [comment]
the code lgtm after the changes, I am not as familiar with uniswap forks so I will approve and leave it to you if you want to keep looking at that or merge and look...

**Review by dmelotik — APPROVED** [review]

**Pull request merged**
